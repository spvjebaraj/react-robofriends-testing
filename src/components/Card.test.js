import React from "react";
import { shallow } from "enzyme";
import Card from "./Card";

it("expect to render Card compoent", () => {
  expect(shallow(<Card />).length).toEqual(1);
});

it("expect to render Card compoent - Snapshot", () => {
  expect(shallow(<Card />)).toMatchSnapshot();
});
