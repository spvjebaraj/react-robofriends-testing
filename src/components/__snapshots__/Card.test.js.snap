// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Card compoent - Snapshot 1`] = `ShallowWrapper {}`;
